src/CMakeLists.txt

```diff
@@ -117,6 +117,47 @@ install( TARGETS gauxc ${GAUXC_EXPORT_DEPENDENCIES}
 
 set_target_properties( gauxc PROPERTIES EXPORT_NAME gauxc )
 
+set(export_properties
+  # currently configurable properties 
+  "GAUXC_ENABLE_TESTS"
+  "GAUXC_ENABLE_HOST"
+  "GAUXC_ENABLE_CUDA"
+  "GAUXC_ENABLE_HIP"
+  "GAUXC_ENABLE_MAGMA"
+  "GAUXC_ENABLE_CUTLASS"
+  "GAUXC_ENABLE_NCCL"
+  "GAUXC_ENABLE_MPI"
+  "GAUXC_ENABLE_OPENMP"
+  "GAUXC_ENABLE_GAU2GRID"
+  "GAUXC_ENABLE_HDF5"
+  "GAUXC_ENABLE_FAST_RSQRT"
+  "GAUXC_BLAS_PREFER_ILP64"
```

Comment:
I can see @loriab's concerns with variabled like this... For now, I just wouldn't export it.